src/pages/PingDetail.svelte

```diff
@@ -129,6 +129,19 @@
           </a>
         </td>
       </tr>
+      <td>
+        Data Catalog
+        <HelpHoverable
+          content={"View this table in Mozilla's instance of DataHub"}
+        />
+      </td>
+      <td>
+        <AuthenticatedLink
+          href={`https://mozilla.acryl.io/dataset/urn:li:dataset:(urn:li:dataPlatform:bigquery,moz-fx-data-shared-prod.${selectedAppVariant.table},PROD)`}
```

Comment:
Since this is on a ping page, shouldn't this link directly to the ping entry? 
```suggestion
          href={`https://mozilla.acryl.io/dataset/urn:li:dataset:(urn:li:dataPlatform:Glean,moz-fx-data-shared-prod.${params.app}.${params.ping},PROD)`}
```
(or something like that)

Reply:
good catch! thanks!